### PR TITLE
feat: split fox discount copies by feature

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - makes things look better for the user by not having "Trade" everywhere including in LP land.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- relates to https://github.com/shapeshift/web/issues/6325

Low - ensure copies still look sane

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing

- Copies correctly convey the "Trade" / "LP Deposit" terminology

### Engineering

- ^

### Operations

- ^

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6453** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6453?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by 0xApotheosis — APPROVED** [review]
Does what it says on the box, two non-blocking comments.

File: src/components/FeeExplainer/FeeExplainer.tsx [hunk 1]
```diff
@@ -337,6 +337,24 @@ type FeeExplainerProps = CardProps & {
...
+  const feature = translate(FEE_MODEL_TO_FEATURE_NAME[props.feeModel])
+  const simulateBodyTranslation: TextPropTypes['translation'] = useMemo(
+    () => [
+      'foxDiscounts.simulateBody',
+      {
+        feature,
+        // Only lowercase the feature if it's a one-word one e.g trade
+        // Assume multiple words should keep their capitalization to keep things simple and avoid more translation strings
+        featureLowerCase:
+          feature.split(' ').length > 1 && feature !== feature.toLowerCase()
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/components/FeeExplainer/FeeExplainer.tsx]: da289a8501

File: src/components/FeeModal/FeeBreakdown.tsx [hunk 2]
```diff
@@ -55,11 +58,20 @@ export const FeeBreakdown = ({
     <Stack spacing={0}>
       <Stack spacing={2} px={8} pt={8} mb={8}>
         <Heading as='h5'>{translate('foxDiscounts.breakdownHeader')}</Heading>
-        <RawText color='text.subtle'>{translate('foxDiscounts.breakdownBody')}</RawText>
+        <RawText color='text.subtle'>
+          {translate('foxDiscounts.breakdownBody', {
+            // Only lowercase the feature if it's a one-word one e.g trade
+            // Assume multiple words should keep their capitalization to keep things simple and avoid more translation strings
+            featureLowerCase:
+              feature.split(' ').length > 1 && feature !== feature.toLowerCase()
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/components/FeeModal/FeeBreakdown.tsx]: da289a8501

**Pull request merged**